packages/edit-site/src/components/header/style.scss

```diff
@@ -5,39 +5,27 @@
 	height: $header-height;
 	box-sizing: border-box;
 	width: 100%;
+	justify-content: space-between;
 
 	@include break-medium() {
-		padding-left: 60px;
-		transition: padding-left 20ms linear;
-		transition-delay: 80ms;
-		@include reduce-motion("transition");
+		body.is-fullscreen-mode & {
+			padding-left: 60px;
+			transition: padding-left 20ms linear;
+			transition-delay: 80ms;
+			@include reduce-motion("transition");
+		}
+
 	}
 
 	.edit-site-header_start,
 	.edit-site-header_end {
-		flex: 1 0;
 		display: flex;
 	}
 
-	@include break-medium() {
-		.edit-site-header_start {
-			// Flex basis prevents the header_start toolbar
-			// from collapsing when shrinking the viewport.
-			flex-basis: calc(#{$header-toolbar-min-width} - #{$header-height});
-		}
-
-		.edit-site-header_end {
-			// Flex basis prevents the header_end toolbar
-			// from collapsing when shrinking the viewport
-			flex-basis: $header-toolbar-min-width;
```

Comment:
I think this is the only place that variable is used. Which means it can be removed from the _variables.scss stylesheet in base-styles now. See also #26827.